### PR TITLE
chore(server-patch): remove server polyfills

### PR DESCRIPTION
**Pull request by manekinekko**
chore(server-patch): remove server polyfill and rename "server-patch" to "univeral-polyfill".
fixing https://github.com/angular/universal/issues/193

### CODE REVIEW

File: modules/universal/server/src/universal-polyfill.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+export * from 'es6-promise';
+export * from 'es6-shim';
 // typescript emit metadata
-import 'reflect-metadata';
+export * from 'reflect-metadata';
 // zone.js to track promises
-import 'zone.js/dist/zone-microtask';
-import 'zone.js/dist/long-stack-trace-zone';
+export * from 'zone.js/dist/zone-microtask';
+export * from 'zone.js/dist/long-stack-trace-zone';
```
Comment (gdi2290) [inline modules/universal/server/src/universal-polyfill.ts]: keep the `import` and rename the file to `universal-polyfills.ts`

**Comment by manekinekko** [comment]
should this one be rebased too ?

**Comment by gdi2290** [comment]
ya

**Comment by jeffwhelpley** [comment]
@manekinekko any idea why we got this automatic message? which author of the commits may they be referring to?

**Comment by manekinekko** [comment]
@jeffwhelpley I think this is because I did a merge instead of rebasing. I closed this PR because of this mistake and I created another one: https://github.com/angular/universal/pull/230

**Pull request closed without merging**